src/main/java/at/hannibal2/skyhanni/features/dungeon/DungeonShadowAssassinNotification.kt

```diff
@@ -0,0 +1,33 @@
+package at.hannibal2.skyhanni.features.dungeon
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.TitleManager
+import at.hannibal2.skyhanni.events.PacketEvent
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.SoundUtils
+import net.minecraft.network.play.server.S44PacketWorldBorder
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import java.lang.reflect.Field
+import kotlin.time.Duration.Companion.seconds
+
+class DungeonShadowAssassinNotification {
+    @SubscribeEvent
+    fun onWorldBoarderChange(event: PacketEvent.ReceiveEvent) {
+        if (!LorenzUtils.inSkyBlock || !LorenzUtils.inDungeons) return
+        if (!SkyHanniMod.feature.dungeon.shadowAssassinJumpNotifier) return
+        //F7 and maybe newer, currently not released, floors have shadow assassins, so they are allowed to be listed here
+        val disable = if (DungeonAPI.dungeonFloor == null) true else DungeonAPI.dungeonFloor!!.contains("7")
+        if (DungeonAPI.inBossRoom && disable) return
```

Comment:
This variable is poorly named (disable does not disable anything on its own). Also please do not use the not null assertion command: 
```suggestion
        if (DungeonAPI.dungeonFloor?.contains("7") == true && DungeonAPI.inBossRoom) return
```